1.1/spec/11-Part-08.adoc

```diff
@@ -71,6 +71,15 @@ geo:GeometryCollection
 
 The restriction imposed on the more general <<Class: SpatialObjectCollection, Spatial Object Collection>> that defines this class is that only instances of <<Class: Geometry, Geometry>> are allowed to be members of it and these are to be indicated with the `rdfs:member` property.
 
+[NOTE]
+====
+Some geometry literal formats, such as WKT, GML and GeoJSON, allow for collections of geometries. In WKT's case, this is indicated with the geometry type _GeometryCollection_. While the names of these objects are the same and this class' and the concepts are similar, there is no RDF/ontology relationship between literals of this type and this class. Since this class contains whole `geo:Geometry` instances, they may have much more information than geometry serialization within a literal geometry collection. 
+
+GeoSPARQL encourages the use of the class `geo:GeometryCollection` in situations where this class and a geometry collection literal seem to be equivalent.
```

Comment:
How can this recommendation be explained? GeoSPARQL is based on the SF specification, which defines GeometryCollection. So sf:GeometryCollection already is an integral part of GeoSPARQL. Also, spatial data engines and GeoSPARQL functions are based on SF. That means that sf:GeometryCollections can act as input or output of GeoSPARQL functions. An RDF collection can not do that. I feel that rather than recommending one or the other we should explain how each type of collection can be used.

Reply:
As above comment.